editor/src/components/canvas/canvas-strategies/strategies/grid-reparent-strategies.spec.browser2.tsx

```diff
@@ -847,6 +847,8 @@ describe('grid reparent strategies', () => {
           backgroundColor: '#f0f',
           width: 79,
           height: 86,
+		  gridColumn: 3,
+		  gridRow: 1,
```

Comment:
does this mean we lost reorder-only reparent? the test is named "...with reorder (no explicit gridRow/gridColumn props"